doc/tutorial/mpi.rst

```diff
@@ -64,8 +64,8 @@ cells over 5 nodes, simulating 100 cells each.
 
 .. code-block::
 
-   srun -n NRANKS python network_ring_mpi.py
-   mpirun -n NRANKS python network_ring_mpi.py
+   srun -n NRANKS python mpi.py
```

Comment:
_also_ ``srun`` is specific to SLURM, not MPI, which might cause quite some confusion on a laptop w/o a resource manager.

Reply:
Aight.